### ISSUE TITLE
File icons (saved) are invisible on dark mode

### ISSUE THREAD

**Issue by hyoretsu** [notepad-plus-plus/notepad-plus-plus#11650]
### Description of the Issue
What should have a saved icon is simply blank in dark mode.

### Steps to Reproduce the Issue
1.  Enable dark mode

### Expected Behavior
We should see a blue diskette just like in light mode.

### Actual Behavior
We instead see nothing, an empty space.

### Debug Information
Notepad++ v8.3.3   (64-bit)
Build time : Mar 13 2022 - 17:20:02
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : E:\My Drive\Backups\Notepad++
OS Name : Windows 11 (64-bit) 
OS Version : 2009
OS Build : 22000.613
Current ANSI codepage : 1252
Plugins : AutoSave.dll NppSaveAsAdmin.dll RunMe.dll 

Light mode:
![image](https://user-images.githubusercontent.com/20804322/167337127-8bfc4ac5-a96e-49ff-8b05-a70dbed8bfee.png)
Dark mode:
![image](https://user-images.githubusercontent.com/20804322/167337099-5146a380-983d-4f2a-bda3-f80a56d0aa63.png)

**Comment by alankilborn** [comment]
No, it's by design that you see nothing here.

**Comment by hyoretsu** [comment]
Weird design for sure, simply removing an icon (only saved) and not the space it occupies.

**Comment by alankilborn** [comment]
I agree with your points; I made some of the same observations when this was originally implemented.

**Comment by hyoretsu** [comment]
Yeah, it's the first time I've seen a dark mode **change** a part of the UI instead of just darkening colors like a theme.
